discovery-provider/plugins/pedalboard/docker/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM node:20-alpine AS base
+
+ARG app_name
+
+# set as env so we dont have to reset arg after multistage build
+ENV APP_NAME=${app_name}
+
+FROM base AS builder
+
+RUN echo "building ${APP_NAME}"
+
+RUN apk add --no-cache libc6-compat
+RUN apk update
+
+WORKDIR /app
+RUN yarn global add turbo
+
+COPY . .
+RUN turbo prune "--scope=${APP_NAME}" --docker
+
+# I should be able to just use base and copy exactly what I need here
+# Copying files doesn't work as expected though so just composing the build
+# for now
+FROM builder AS installer
```

Comment:
i could not find a way to get this to work without using builder as the base for installer